app/styles/_top.scss

```diff
@@ -365,7 +363,7 @@ div.outer-container {
                             margin-right: auto;
                             margin-top: 22px;
                             position: relative;
-                            width: 516px;
+                            width: 294px;
```

Comment:
As we removed video from home page, change the width to the same as the download button so it will be horizontally centered.